agenda/presentation/src/main/java/com/pronoidsoftware/agenda/presentation/overview/model/AgendaOverviewItemUi.kt

```diff
@@ -1,52 +1,59 @@
 package com.pronoidsoftware.agenda.presentation.overview.model
 
-import androidx.compose.ui.graphics.Color
-import com.pronoidsoftware.agenda.domain.model.AgendaItemType
-import com.pronoidsoftware.core.presentation.designsystem.TaskyBlack
-import com.pronoidsoftware.core.presentation.designsystem.TaskyBrown
-import com.pronoidsoftware.core.presentation.designsystem.TaskyDarkGray
-import com.pronoidsoftware.core.presentation.designsystem.TaskyGreen
-import com.pronoidsoftware.core.presentation.designsystem.TaskyLightGray
-import com.pronoidsoftware.core.presentation.designsystem.TaskyLightGreen
-import com.pronoidsoftware.core.presentation.designsystem.TaskyWhite
+sealed class AgendaOverviewItemUi(
+    open val id: String,
+) {
+    data class TimeMarker(
+        override val id: String = "TimeMarker",
+    ) : AgendaOverviewItemUi(id = id)
+
+    data class Item(
+        val item: AgendaOverviewItemContents,
+        override val id: String,
+    ) : AgendaOverviewItemUi(id = id)
+}
```

Comment:
The ID doesn't need to be overrideable here, right? This would also do it:
```
sealed class AgendaOverviewItemUi(
   val id: String,
) {
    data object TimeMarker : AgendaOverviewItemUi(id = "TimeMarker")

    data class Item(
        val item: AgendaOverviewItemContents,
    ) : AgendaOverviewItemUi(id = item.id)
}
```

Reply:
yup, nice catch, thanks!